hicommonwealth/commonwealth
# 5426 - Overview Page Should Filter Out Archived and Spammy Threads

Currently, the Overview page shows both archived and spam threads. This PR removes them from the page.

(note that this is branched off of #3924. The only relevant file to check for this PR is: `packages/commonwealth/client/scripts/views/pages/overview/index.tsx`)

## Link to Issue
Closes: #5426 

## Description of Changes
- provides additional filtering such that the Overview page only shows recent threads that are not archived or spam. 

## Test Plan
- visit overview page
- select a thread and mark it as spam or archive it
- navigate back to the overview page. Aforementioned thread should no longer be present on the page. 

## Other Considerations
- This is a fast follow for PR #4338 (adds ability for moderator to archive thread).
